binaries/scheduler/main.go

```diff
@@ -17,21 +19,22 @@ func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	parser := config.DefaultParser()
-	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
-	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
-	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	env, o := config.Default()
+	o["Workers"][""] = &config.WorkersThriftConfig{Type: "rpc"}
+	o["Cluster"][""] = &local.ClusterLocalConfig{Type: "local"}
 
-	// Construct scootapi server handler based on config.
-
-	handler, err := parser.Create([]byte(*cfgText))
+	// Construct scootapi server handler based on configuration and options.
+	var handler scoot.CloudScoot
+	err := ice.ParseAndEval(env, o, *cfgText, &handler)
 	if err != nil {
 		log.Fatal("Error configuring Scoot API: ", err)
 	}
 
 	// Start API Server
 	log.Println("Starting API Server")
 
+	log.Println("Could this possible work?!", handler)
```

Comment:
Yes, it is, because this is currently not ready for submission, as I mentioned in the PR description:
It creates the scheduler, but not the health-checker/stats handler, which should be easy to put in if we like the approach.